mflowgen/common/soc-rtl-v2/run.sh

```diff
@@ -6,6 +6,10 @@ CUR_DIR=$(pwd)
 # Clone the SoC Repo
 git clone https://github.com/StanfordAHA/AhaM3SoC.git aham3soc
 
+# We'll need this until the new PHY RDL is incorporated and working (right?);
+# then we can delete it before the final merge(s)
+(cd aham3soc; git checkout phy_pads)
```

Comment:
Since we have all the collateral from them now, we should put it in place and merge the phy_pads branch.

Reply:
I see that you approved my aham3soc merge finally :)
So I will undo this hack and start a run to verify that it works (see upcoming to-do list).